besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -767,6 +767,14 @@ static class JsonRPCHttpOptionGroup {
         description =
             "Specifies the maximum number of requests in a single RPC batch request via RPC. -1 specifies no limit  (default: ${DEFAULT-VALUE})")
     private final Integer rpcHttpMaxBatchSize = DEFAULT_HTTP_MAX_BATCH_SIZE;
+
+    @CommandLine.Option(
+        names = {"--rpc-http-max-resource-intensive-per-batch-size"},
+        paramLabel = MANDATORY_INTEGER_FORMAT_HELP,
+        description =
+            "Specifies the maximum number of resource-intensive requests in a single RPC batch request via RPC. -1 specifies no limit  (default: ${DEFAULT-VALUE})")
```

Comment:
For the UX will be nice to have a way to know which are the resource intensive methods